plugins/modules/s3_object.py

```diff
@@ -3,6 +3,7 @@
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import (absolute_import, division, print_function)
+from curses import KEY_PRINT
```

Comment:
is this intentional?

Reply:
Got added while rebasing. I shall remove this.